src/codeu/chat/server/Controller.java

```diff
@@ -39,6 +41,9 @@ public Controller(Uuid serverId, Model model) {
     this.uuidGenerator = new RandomUuidGenerator(serverId, System.currentTimeMillis());
   }
 
+  public void addMemberToConversation(Uuid conversationID, Uuid owner, Uuid member) {
+    ConversationHeader.members.add(member);
```

Comment:
You need to get an instance of conversation from the `model`, you are just accessing the class here.